sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,593 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  WebResource,
+  ServiceClient,
+  ServiceClientOptions,
+  HttpOperationResponse,
+  AtomXmlOperationSpec,
+  signingPolicy,
+  userAgentPolicy,
+  logPolicy,
+  proxyPolicy,
+  atomSerializationPolicy,
+  RequestPolicyFactory,
+  AtomXmlSerializer,
+  URLBuilder,
+  ProxySettings
+} from "@azure/core-http";
+
+import { httpAtomXml } from "./log";
+import { SasServiceClientCredentials } from "./util/sasServiceClientCredentials";
+import * as Constants from "./util/constants";
+
+import { QueueResourceSerializer, QueueOptions } from "./serializers/queueResourceSerializer";
+import { TopicResourceSerializer, TopicOptions } from "./serializers/topicResourceSerializer";
+import {
+  SubscriptionResourceSerializer,
+  SubscriptionOptions
+} from "./serializers/subscriptionResourceSerializer";
+import { RuleResourceSerializer, RuleOptions } from "./serializers/ruleResourceSerializer";
+
+import {
+  ListRequestOptions,
+  QueueResult,
+  ListQueuesResult,
+  TopicResult,
+  ListTopicsResult,
+  SubscriptionResult,
+  ListSubscriptionsResult,
+  RuleResult,
+  ListRulesResult
+} from "./managementOperationInterfaces";
```

Comment:
All the interfaces from the `managementOperationInterfaces.ts` file are used only in this current file. Please consider moving them here.

Reply:
Other additional changes w.r.t the interfaces that we discussed offline, is about having different names for the individual operation response types, so that they can be updated with minimal impact to users in terms of backward compatibility.

@ramya-rao-a @chradek @bterlson